docs/sources/_index.md

```diff
@@ -17,22 +17,22 @@ title: Grafana documentation
 ## Installing Grafana
 
 <div class="nav-cards">
-    <a href="{{< relref "installation/debian.md" >}}" class="nav-cards__item nav-cards__item--install">
+    <a href="{{< relref "setup-grafana/installation/debian/" >}}" class="nav-cards__item nav-cards__item--install">
```

Comment:
Perhaps this is out of scope for this PR, but I do not favor `setup` here in noun form when it is really supposed to be a verb.

Reply:
Please consider changing this to `configure` instead of `setup` for two reasons: it is the wrong part of speech, and it does not properly convey what a user needs to do.

Reply:
This will get addressed in a subsequent PR.